pkg/services/searchV2/object/testdata/dash_raw.jsonc

```diff
@@ -9,7 +9,7 @@
 //  | Type: []string                                          | Type: []string | Type: []int64 | Type: []string                   |
 //  +---------------------------------------------------------+----------------+---------------+----------------------------------+
 //  | alerting/testdata_alerts.json                           |                | 16402         | a3679c3611fc3a83b0a9418e8f4cd0eb |
-//  | all-panels.json                                         |                | 23018         | a52c6dab16ecc2d3fd0b1e1edad930d6 |
+//  | all-panels.json                                         |                | 23019         | 43e90576e39fbc52a0560474489c5671 |
```

Comment:
Do we know why this changed? It is the test for `github.com/grafana/grafana/pkg/services/searchV2/object` that is failing, which uses this file. Could you revert the change to this file and try pushing again to see if the test passes? @gtk-grafana